### PR TITLE
Added possibility to force HTTPS. 

### PR DESCRIPTION
**Pull request by cinqi**
Can be used for example when web server is behind load balancer and SSL is terminated in load balancer.

### CODE REVIEW

File: cdn.php [hunk 1]
```diff
@@ -43,7 +43,9 @@ public function onOutputGenerated()
         }
 
         // set the protocol to HTTPS if you access that way
-        if (isset($_SERVER['HTTPS']) && strtolower($_SERVER['HTTPS']) == 'on') {
+        if ( (isset($_SERVER['HTTPS']) && strtolower($_SERVER['HTTPS']) == 'on') ||
+           ( isset($config['forcehttps']) && $config['forcehttps'] == 1 ))
```
Comment (rhukster) [inline cdn.php]: I think this should be:

```
 isset($config['forcehttps']) && $config['forcehttps'] == true )
```

as these settings are typically true/false and the blueprint being `validate: type: bool` is setting this to be true/false

**Review by rhukster — CHANGES_REQUESTED** [review]
See inline comments, but its a minor thing :)

File: cdn.php (hunk 1 above)
Comment (cinqi) [reply cdn.php]: I'll take a look at it. 

Originally I wrote it like this:

`isset($config['forcehttps']) && $config['forcehttps'] )`

but that didn't work, thus == 1. I'll take a look at if == true works.

Comment (cinqi) [reply cdn.php]: == true works, pushed new version to the branch.

**Review by rhukster — APPROVED** [review]

**Pull request merged**